spec/03-plugins/03-http-log/01-log_spec.lua

```diff
@@ -269,4 +312,67 @@ for _, strategy in helpers.each_strategy() do
       end, 10)
     end)
   end)
-end
+
+  describe("Plugin: http-log (log) [#" .. strategy .. "]", function()
+    local proxy_client
+
+    setup(function()
+      local bp = helpers.get_db_utils(strategy)
+
+      bp.plugins:insert {
+        name     = "http-log",
+        config   = {
+          http_endpoint = "http://" .. helpers.mock_upstream_host
+                                    .. ":"
+                                    .. helpers.mock_upstream_port
+                                    .. "/post_log/http"
+        }
+      }
+
+      assert(helpers.start_kong({
+        database = strategy,
+        nginx_conf = "spec/fixtures/custom_nginx.template",
+      }))
+    end)
+
+    teardown(function()
+      helpers.stop_kong()
+    end)
+
+    before_each(function()
+      proxy_client = helpers.proxy_client()
+    end)
+
+    after_each(function()
+      if proxy_client then
+        proxy_client:close()
+      end
+    end)
+
+    it("successfully executes globally when route does not exist", function()
+    	-- setup: cleanup logs
+      local test_error_log_path = helpers.test_conf.nginx_err_logs
```

Comment:
We should not do this ourselves, a previous call to `stop_kong` (from the sibling describe block for example) should take care of it for us.

Reply:
So i just committed what I think you wanted, basically don't do the cleanup part at the top anymore but just reference the helpers nginx_err_logs down below. That's what you wanted right?